Purchases/Purchasing/RCReceiptFetcher.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  RCReceiptFetcher.swift
+//  Purchases
+//
+//  Created by Javier de Martín Gil on 8/7/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc open class RCReceiptFetcher: NSObject {
+
+    @objc open func receiptData() -> Data? {
+
+        guard let receiptURL: URL = Bundle.main.appStoreReceiptURL else {
+            Logger.debug(Strings.receipt.no_sandbox_receipt_restore)
+            return nil
+        }
+
+        #if TARGET_OS_WATCH
```

Comment:
I think this might not work in Swift - xcode won't complain, but the pre-processor macro isn't defined, so it will always evaluate to false, even on watch. 

The swift equivalent of this would be
```swift
#if os(watchOS)
```

see this [article for more info](https://nshipster.com/swift-system-version-checking/)

Reply:
Yup totally correct, I missed that. `APITester` target didn't complain and compiled but it'll probably evaluate to false on watchOS. Thanks!